### PR TITLE
Buff magic weapon, enlarge/reduce, and lightning lure (SP 20, 26–27)

### PR DESCRIPTION
**Pull request by JamesMcClung**
Note that the enlarge/reduce and lightning lure buffs are slightly different from the original spell proposals (which have since been edited so they currently do match)

### CODE REVIEW

File: spells.yml [hunk 1]
```diff
@@ -3532,9 +3532,9 @@ enlarge_reduce:
...
 
-    **Enlarge.** The target's size doubles in all dimensions, and its weight is multiplied by eight. This growth increases its size by one category—from Medium to Large, for example. If there isn't enough room for the target to double its size, the creature or object attains the maximum possible size in the space available. Until the spell ends, the target also has advantage on Strength checks and Strength saving throws. The target's weapons also grow to match its new size. While these weapons are enlarged, the target's attacks with them deal 1d4 extra damage.
+    **Enlarge.** The target's size doubles in all dimensions, and its weight is multiplied by eight. This growth increases its size by one category—from Medium to Large, for example. If there isn't enough room for the target to double its size, the creature or object attains the maximum possible size in the space available. Until the spell ends, the target's reach increases by 5 feet and it has advantage on Strength checks and Strength saving throws. The target's weapons also grow to match its new size. While these weapons are enlarged, the target's attacks with them deal 1d4 extra damage.
```
Comment (bwhetherington) [inline spells.yml:3535]: Grammar nit: there should be a comma after "increases by 5 feet".

File: spells.yml [hunk 2]
```diff
@@ -3532,9 +3532,9 @@ enlarge_reduce:
...
 
-    **Reduce.** The target's size is halved in all dimensions, and its weight is reduced to one-eighth of normal. This reduction decreases its size by one category—from Medium to Small, for example. Until the spell ends, the target also has disadvantage on Strength checks and Strength saving throws. The target's weapons also shrink to match its new size. While these weapons are reduced, the target's attacks with them deal 1d4 less damage (this can't reduce the damage below 1).
+    **Reduce.** The target's size is halved in all dimensions, and its weight is reduced to one-eighth of normal. This reduction decreases its size by one category—from Medium to Small, for example. Until the spell ends, the target's reach decreases by 5 feet (to a minimum of 5 feet) and it has disadvantage on Strength checks and Strength saving throws. The target's weapons also shrink to match its new size. While these weapons are reduced, the target's attacks with them deal 1d4 less damage (this can't reduce the damage below 1).
```
Comment (bwhetherington) [inline spells.yml:3537]: Once again, there should be a comma after "(to a minimum of 5 feet)".

File: spells.yml [hunk 3]
```diff
@@ -6391,13 +6392,13 @@ lightning_lure:
   components: V
   concentration: false
   description: |-
-    You create a lash of lightning energy that strikes at one creature of your choice that you can see within 15 feet of you. The target must succeed on a Strength saving throw or be pulled up to 10 feet in a straight line toward you and then take 1d8 lightning damage if it is within 5 feet of you.
+    You create a lash of lightning energy that strikes at one creature of your choice that you can see within range. The target must succeed on a Strength saving throw or take 1d10 lightning damage and be pulled up to 10 feet in a straight line toward you. On a successful save, the target takes half as much damage and isn't pulled. If you pull the target to a space within 5 feet of you, its speed is reduced by 10 feet until the end of your next turn.
```
Comment (bwhetherington) [inline spells.yml:6395]: Comma

**Review by bwhetherington — CHANGES_REQUESTED** [review]
Looks good, barring some missing commas.

**Comment by JamesMcClung** [comment]
> Looks good, barring some missing commas.

I believe it is actually correct to not have those commas

File: spells.yml (hunk 3 above)
Comment (bwhetherington) [reply spells.yml:6395]: This one does not need a comma, but the others do.

**Comment by bwhetherington** [comment]
You have independent clauses connected by "and". A comma is necessary there, and this is also the standard followed by the rest of the spells, including the rest of the descriptions of these spells. Look at _enlarge_ as an example:

> The target's size doubles in all dimensions, and its weight is multiplied by eight.

If you have a sentence that goes something like "It <does something> and it <does something else>", then you need a comma before the "and". See [here](https://www.brandeis.edu/writing-program/resources/students/academic/style-grammar/comma-rules.html#:~:text=Use%20commas%20to%20separate%20independent,but%20my%20brother%20prefers%20chocolate.) for more specifics.

**Comment by JamesMcClung** [comment]
Normally yes, but the presence of the subordinate clause `Until the spell ends` might change that. I'm still looking for an answer online

**Review by bwhetherington — APPROVED** [review]

**Pull request merged**